src/recommendations/RecommendationsPage.jsx

```diff
@@ -47,7 +50,7 @@ const RecommendationsPage = (props) => {
     }
   };
 
-  if (!isLoading && recommendations.length < 3) {
+  if (!isLoading && recommendations.length < RECOMMENDATIONS_COUNT) {
```

Comment:
Don't we want to show 3 recommendations?

Reply:
I am not sure where this `3` came from so for safe side I have used `RECOMMENDATIONS_COUNT` and have also created a thread with UX for confirmation. They are finalizing a strategy where we will backfill the missing recommendations with general ones but for now lets keep it the condition as mentioned above.